get_rtma

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
Would you please make this a `/bin/bash` script explicitly. Our cloud work lately has led me to realize that the default `/bin/sh` is not always `bash` and we do rely on that functionality heavily.

Reply:
No problem. Good to know this.